sql/core/src/test/resources/sql-tests/results/subquery/scalar-subquery/scalar-subquery-count-bug.sql.out

```diff
@@ -86,14 +86,14 @@ from l
 -- !query schema
 struct<a:int,b:decimal(2,1),scalarsubquery(a):boolean>
 -- !query output
-1	2.0	NULL
-1	2.0	NULL
+1	2.0	false
+1	2.0	false
```

Comment:
@jchen5 was the old result wrong?

Reply:
Yes, old result here was wrong, new result is correct.

@Hisoka-X There's a comment about it in the input file, please update it to remove the "but this case is wrong due to bug SPARK-43156" part

Reply:
Done